pkg/notes/notes.go

```diff
@@ -849,3 +879,52 @@ func prettifySIGList(sigs []string) string {
 
 	return sigList
 }
+
+// ApplyMap Modifies the content of the release using information from
+//  a ReleaseNotesMap
+func (rn *ReleaseNote) ApplyMap(noteMap *ReleaseNotesMap) error {
+
+	if noteMap.ReleaseNote.Author != nil {
+		rn.Author = *noteMap.ReleaseNote.Author
+		rn.AuthorURL = "https://github.com/" + *noteMap.ReleaseNote.Author
+	}
+
+	if noteMap.ReleaseNote.Text != nil {
+		rn.Text = *noteMap.ReleaseNote.Text
+	}
+
+	if noteMap.ReleaseNote.Documentation != nil {
+		rn.Documentation = append(rn.Documentation, *noteMap.ReleaseNote.Documentation...)
```

Comment:
I expect that we overwrite here rather than appending, wdyt?

Reply:
Yes it makes more sense. If you want to append data you can still copy the original value. I changed all slice values so that they overwrite the originals to get a consistent behavior